README.md

```diff
@@ -228,20 +228,114 @@ Now everything has been set up and OP-TEE is ready to be used.
 ---
 ### 4.3 Juno
 Juno has been supported in OP-TEE since mid October 2014.
-#### 4.3.1 Get the compiler and source
-Follow the instructions in the "4.1 Basic setup".
 
-#### 4.3.2 Build
+WARNING: 
```

Comment:
Not really clear. type I and type II of optee_linuxdriver is not clear.

I would simply say that this script provides a coherent set of components (optee / Linux Kernel / ...), and that further release will align Juno setup with other platforms setup.